### ISSUE TITLE
require-description error message is unclear

### ISSUE THREAD

**Issue by lennyburdette** [dotansimha/graphql-eslint#924]
**Describe the bug**
The error message for `require-description` provides only the line/column and the GraphQL kind. It would be way easier to fix the issue if the message included the name of the element! (Especially when using a code-first approach, or when combining multiple graphql files into a single file before linting — in those cases, the line numbers are not helpful!)

```
6:6  error  Description is required for nodes of type "ObjectTypeDefinition"  @graphql-eslint/require-description
```

**Expected behavior**
The error message could say

```
6:6  error  Description is required for `type Query`     @graphql-eslint/require-description
8:6  error  Description is required for `Query.products` @graphql-eslint/require-description
```

**Environment:**

- OS: any
- `@graphql-eslint/eslint-plugin`: 3.6.0
- NodeJS: 16.x

**Comment by B2o5T** [comment]
I like your proposal 👍 report message will be clarified in the next version for this rule 🚀

**Comment by dotansimha** [comment]
Fixed in `@graphql-eslint/eslint-plugin@3.8.0` 🎉
